### PR TITLE
Merge 126.0.0 alpha.3 tag

### PR DESCRIPTION
**Pull request by aprucolimartins**
MetalK8s 126.0.0-alpha.3
========================

MetalK8s 126.0.0-alpha.3 is a patch release (see
[the main release 126.0.0](https://github.com/scality/metalk8s/releases/126.0.0)).

Useful links
------------

- [Documentation](https://metal-k8s.readthedocs.io/en/126.0.0-alpha.3)
- [Upgrade notes](https://metal-k8s.readthedocs.io/en/126.0.0-alpha.3/operation/upgrade.html)
- [Changelog](https://github.com/scality/metalk8s/blob/126.0.0-alpha.3/CHANGELOG.md)

What's new
----------

- Salt Master worker_threads and timeout are now configurable

- Alert re-push mechanism works properly

- Versions is properly shown on UI after upgrade

- Prevent Python 3.6 conflicts with newer versions of OpenStackSDK

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,8 +1,20 @@
...
+
+- Alert re-push mechanism works properly
+  (PR[#4141](https://github.com/scality/metalk8s/pull/4141))
+
+- Versions is properly shown on UI after upgrade
+  (PR[#4140](https://github.com/scality/metalk8s/pull/4140))
+
+- Prevent Python 3.6 conflicts with newer versions of OpenStackSDK
+  (PR[#4139](https://github.com/scality/metalk8s/pull/4139))
+
```
Comment (TeddyAndrieux) [inline CHANGELOG.md]: This line should be removed it's not supposed to be show to the user, it's only for CI

File: CHANGELOG.md [hunk 2]
```diff
@@ -28,6 +40,18 @@
...
+    (PR[#4148](https://github.com/scality/metalk8s/pull/4148))
+
+  - Bump Docker login-action to [3](https://github.com/docker/login-action/tree/v3.0.0)
+    (PR[#4147](https://github.com/scality/metalk8s/pull/4147))
+
+  - Bump Docker setup-buildx-action to [3](https://github.com/docker/setup-buildx-action/tree/v3.0.0)
+    (PR[#4146](https://github.com/scality/metalk8s/pull/4146))
+
+  - Bump Scality Actions to [1.6.0](https://github.com/scality/actions/tree/1.6.0)
+    (PR[#4150](https://github.com/scality/metalk8s/pull/4150))
```
Comment (TeddyAndrieux) [inline CHANGELOG.md]: Same all those should be removed

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,8 +1,22 @@
 # CHANGELOG
+
 ## Release 126.0.0 (in development)
+## Release 126.0.0-alpha.3
```
Comment (TeddyAndrieux) [inline CHANGELOG.md]: This line should be removed

**Review by TeddyAndrieux — CHANGES_REQUESTED** [review]
Some changes need to be done on the changelog

File: CHANGELOG.md (hunk 1 above)
Comment (aprucolimartins) [reply CHANGELOG.md]: Done !!!

File: CHANGELOG.md (hunk 2 above)
Comment (aprucolimartins) [reply CHANGELOG.md]: Done !!!

File: CHANGELOG.md (hunk 3 above)
Comment (aprucolimartins) [reply CHANGELOG.md]: Done !!!

**Review by TeddyAndrieux — APPROVED** [review]

**Pull request merged**